### ISSUE TITLE
Betanet account doesn't work with regular account

### ISSUE THREAD

**Issue by lertsoft** [mysteriumnetwork/node#2814]
**Describe the bug**

**To Reproduce**
1. Go to 'https://my.mysterium.network'
2. create an account
3. Go to 'https://betanet.mysterium.network/'
4. try to log in with the account created in step 1
5. See error

**Expected behavior**
I expected to be able to use the same account for both the beta software and the pre-production software.

**Environment (please complete the following information):**
 - Node version: [e.g. 0.40]
 - OS: [e.g. raspbian buster, debian 10]

**Additional context**
While setting up my node I could not use the pre-production software because I did not have 70myst token to send to a wallet require by the software so one of the customer services people gave me a referral for the betanet software to test it on.

**Comment by lertsoft** [comment]
**The comment that I sent to the live chat** 
 I don't know if this is how it is being supposed to work, but once I setup my node I was asked if I wanted to use my mmn API key. which the account on 'https://my.mysterium.network' account has but when I tried setting it up it wouldn't let me.
so I proceeded without that integration to do it later. 
After I made it to the dashboard and later to the settings of the node to revisit my API integration it still would not register. so I click on the link that says to get the API, tried several times to log in to the account in 'https://my.mysterium.network' without any success. then I tried creating a new account but with the same email, username, and same password as the account on the main website and it let me do it. 
Again I don't know if that is intentional or not but I think that might be a bug because now there are two duplicates account for the same thing.
